specs/SetMemoryUsageLevel.md

```diff
@@ -0,0 +1,145 @@
+# Background
+Some WebView2 apps want to continue to run scripts while inactive and therefore cannot make usage of `TrySuspend` and `Resume` APIs to reduce resource consumption.
+We are introducing WebView2 API to reduce memory usage only for this type of inactive WebViews.
+
+# Description
+You may set the `MemoryUsageTargetLevel` property to have the WebView2 consume less memory and going back to normal usage. This is useful when your Win32 app becomes invisible, 
+but still wants to have script running or monitoring requests from network.
+
+# Examples
+## .NET, WinRT
+```c#
+async protected void OnBecomingInactive()
+{
+    // CanSuspendWebPage() checks whether the current web contents in WebView can be suspended.
```

Comment:
CanSuspendWebPage in comment and CanSuspendWebView in code. Also would be good to specify this is an app dependent logic
```suggestion
    // CanSuspendWebPage() uses app specific logic to check whether the current web contents in the WebView2 can be suspended.
```